command/sub8_launch/launch/tf.launch

```diff
@@ -13,7 +13,7 @@
     </node>
 
     <node pkg="tf" type="static_transform_publisher" name="camera_down_broadcaster"
-      args="0.0537  0.1048 -0.2565   7.07106781e-01 -7.07106781e-01 0.0 0.0 base_link down_left_cam 100">
+      args="0.054 0.105 -0.256 1 0 0 0 0 base_link down_left_cam 100">
```

Comment:
Sorry, one zero too many.

Even saying that we know this measurement to the order of millimeters is really pushing it. We should really not use more than four significant figures for any of these measurements